client/me/purchases/remove-purchase/index.jsx

```diff
@@ -212,6 +230,43 @@ class RemovePurchase extends Component {
 		);
 	}
 
+	shouldShowPreCancelationDialog() {
+		const { purchase } = this.props;
+		return (
+			config( 'pre_cancelation_modal_enabled' ) &&
+			isPlan( purchase ) &&
+			! isJetpackPlan( purchase ) &&
+			! isGSuiteOrGoogleWorkspace( purchase )
+		);
+	}
+
+	renderPreCancelationDialog() {
+		const { site, purchase, primaryDomain } = this.props;
+		const primaryDomainName = hasCustomDomain && primaryDomain ? primaryDomain.name : '';
+
+		let wordpressComURL = '';
+		if ( typeof site.wpcom_url !== 'undefined' ) {
+			wordpressComURL =
+				site.wpcom_url.length < 35
+					? site.wpcom_url
+					: site.wpcom_url.substr( 0, 10 ) + '…' + site.wpcom_url.slice( -20 );
+		}
+
+		return (
+			<PreCancelationDialog
+				isDialogVisible={ this.state.isShowingPreCancelationDialog }
+				closeDialog={ this.closeDialog }
+				removePlan={ this.showRemovePlanDialog }
+				site={ site }
+				hasDomain={ hasCustomDomain }
```

Comment:
Like above, `hasCustomDomain` is a function, not a boolean :)

Reply:
this should be fixed, as above